test/functional/mempool_package_onemore.py

```diff
@@ -79,6 +86,10 @@ def run_test(self):
         # and the second chain should work just fine
         self.chain_transaction(self.nodes[0], [second_chain], [0], second_chain_value, fee, 1)
 
+        # But now that we're at the additional carve-out limit, the third base output being spent will be rejected
+        # For both ancestor's descendant limits
+        assert_raises_rpc_error(-26, "too-long-mempool-chain, too many descendants", self.chain_transaction, self.nodes[0], [chain[0][0]], [2], chain[0][1], fee, 1)
```

Comment:
"too-long-mempool-chain, too many descendants" used 5x, could hoist to a `TOO_MANY_DESCENDANTS` constant

Reply:
done